starter-code/index.html

```diff
@@ -2,30 +2,53 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" type="text/css" href="/Users/LeFinancier/code/Semana1/Sabado/lab-css-instagram-clone/starter-code/css/main.css">
 </head>
-<body>
-  Instagram
-
-  Capture and Share the World's Moments
-
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-
-  Oh yeah, did we mention it's free?
-
-  Download on the App Store
-  Get it on Google Play
+<body>
+  <div class="upper">
+      <img src="/Users/LeFinancier/code/Semana1/Sabado/lab-css-instagram-clone/starter-code/images/brand.png" alt="Instagram" class="middle">
```

Comment:
Debes poner los links como ruta `relativa`, eso significa que es relativa al proyecto y a la carpeta donde está contenida y no `estática` que está anclada a tu máquina, de esa forma nadie a demás que tu en tu máquina puede ver los archivos. tendrás que cambiar todos así: 
```suggestion
      <img src="images/brand.png" alt="Instagram" class="middle">
```